### ISSUE TITLE
我就在第一层16通道加了个dca，显存就从6变成了60，直接跑不了。你这个网络的显存需求也太高了，是我的问题还是你这个网络的问题喔

### ISSUE THREAD

**Issue by honghess** [gorkemcanates/Dual-Cross-Attention#12]

**Comment by gorkemcanates** [comment]
DCA should not bring extensive amount of memory. I cant tell for sure without seeing the code.

**Comment by QH-Jia** [comment]
请问你的数据集是如何放置的，在数据加载的部分，是索引文本文件中的图片名吗?

**Comment by QH-Jia** [comment]
> _No description provided._

你好  可以发一下你的数据集是怎么放置的吗，需要建立自己的训练集和测试集的文本文件吗

**Comment by honghess** [comment]
> > _No description provided._
> 
> 你好 可以发一下你的数据集是怎么放置的吗，需要建立自己的训练集和测试集的文本文件吗

inputs//your_dataname//images and masks//0,就是UNet++的框架，unext也是这个框架，需要保证名字是对应的

**Comment by QH-Jia** [comment]
好的，感谢感谢，我已经发现需要修改的地方了，现在已经跑起来了，请问你是用几张卡测试的，中间出现了什么问题呢
---- 回复的原邮件 ----
***@***.***>发送日期2024年7月12日 ***@***.***>***@***.***>,
***@***.***>主题Re: [gorkemcanates/Dual-Cross-Attention] 我就在第一层16通道加了个dca，显存就从6变成了60，直接跑不了。你这个网络的显存需求也太高了，是我的问题还是你这个网络的问题喔 (Issue #12)
No description provided.
你好 可以发一下你的数据集是怎么放置的吗，需要建立自己的训练集和测试集的文本文件吗
inputs//your_dataname//images and masks//0,就是UNet++的框架，unext也是这个框架，需要保证名字是对应的

**Comment by QH-Jia** [comment]
你好，想问一下为什么我的数据集有1000张图片，为什么他这个程序的每个网络测试都只给我测试了50张呢，而且我看网络设置明明测试集设置的是数据集的20%呀，你有遇到这个问题吗？
---- 回复的原邮件 ----
***@***.***>发送日期2024年7月12日 ***@***.***>***@***.***>,
***@***.***>主题Re: [gorkemcanates/Dual-Cross-Attention] 我就在第一层16通道加了个dca，显存就从6变成了60，直接跑不了。你这个网络的显存需求也太高了，是我的问题还是你这个网络的问题喔 (Issue #12)
No description provided.
你好 可以发一下你的数据集是怎么放置的吗，需要建立自己的训练集和测试集的文本文件吗
inputs//your_dataname//images and masks//0,就是UNet++的框架，unext也是这个框架，需要保证名字是对应的

**Comment by honghess** [comment]
> 你好，想问一下为什么我的数据集有1000张图片，为什么他这个程序的每个网络测试都只给我测试了50张呢，而且我看网络设置明明测试集设置的是数据集的20%呀，你有遇到这个问题吗？ ---- 回复的原邮件 ---- ***@***.***>发送日期2024年7月12日 ***@***.***>***@***.***>, ***@***.***>主题Re: [gorkemcanates/Dual-Cross-Attention] 我就在第一层16通道加了个dca，显存就从6变成了60，直接跑不了。你这个网络的显存需求也太高了，是我的问题还是你这个网络的问题喔 (Issue #12) No description provided. 你好 可以发一下你的数据集是怎么放置的吗，需要建立自己的训练集和测试集的文本文件吗 inputs//your_dataname//images and masks//0,就是UNet++的框架，unext也是这个框架，需要保证名字是对应的 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

如果是val的图片数量不对就要看你的那个train_img_ids, val_img_ids = train_test_split(img_ids, test_size=0.2, random_state=41)这个代码怎么设置的了，如果这里和我一样的话那就去看你的数据集放的对不对

**Comment by NetfriendT** [comment]
@honghess 你好，请问可以和你交流一下DAC模块的一些问题吗？

**Comment by honghess** [comment]
> @honghess 你好，请问可以和你交流一下DAC模块的一些问题吗？

ok

**Comment by NetfriendT** [comment]
> > @honghess 你好，请问可以和你交流一下DAC模块的一些问题吗？
> 
> ok



> > @honghess 你好，请问可以和你交流一下DAC模块的一些问题吗？
> 
> ok

方便加个微信吗（KamenRider_PPP），或者这上面交流

**Comment by honghess** [comment]
> 我的问题主要是，我现在使用的是一个mamba的类u-net3d模型，我想知道这个DAC模块在修改后能不能用，毕竟不是cnn提取的特征

就是常规的qkv两个重合一起，你其他的transform能跑他就能跑

**Comment by mrbird643** [comment]
> > 我的问题主要是，我现在使用的是一个mamba的类u-net3d模型，我想知道这个DAC模块在修改后能不能用，毕竟不是cnn提取的特征
> 
> 就是常规的qkv两个重合一起，你其他的transform能跑他就能跑

所以你在使用之后真的很大（带来显存的增加）？ @honghess
